tests/autorestart/test_container_autorestart.py

```diff
@@ -195,27 +195,27 @@ def kill_process_by_pid(duthost, container_name, program_name, program_pid):
                 .format(program_name, container_name))
 
 
-def is_hiting_start_limit(duthost, container_name):
+def is_hiting_start_limit(duthost, service_name, container_name):
     """
     @summary: Determine whether the container can not be restarted is due to
```

Comment:
Can we rephrease `container` ---> `service` since we change the argument name?

Reply:
Updated